bhohnco/kingsCastle
# Feature/search bar component

Is this a fix or a feature?
* feature
What is the change or fix?
* search bar now fetches API data 
* Still needs to display book info after search 
How should this be tested?
* Try to search for a book and check the output in the console. 
Linked Issues: #5 
Add A Reviewer if Necessary
*
